Source/OpenTK/Platform/Android/1.0/Enums.cs

```diff
@@ -0,0 +1,18 @@
+#region --- License ---
+/* Licensed under the MIT/X11 license.
+ * Copyright (c) 2011 Xamarin, Inc.
+ * Copyright 2013 Xamarin Inc
+ * This notice may not be removed from any source distribution.
+ * See license.txt for licensing detailed licensing details.
+ */
+#endregion
+
+using System;
+
+namespace OpenTK.Graphics {
+	public enum GLVersion {
```

Comment:
Sorry, used Xamarin code as is. I will run a cleanup pass, since we probably won't remerge an updated version of their code (hopefully they will switch to this build), this shouldn't be a problem.